Dockerfile

```diff
@@ -59,19 +58,18 @@ RUN apt-get update --assume-yes \
         zlib1g-dev \
         libsdl2-dev \
         mesa-common-dev \
-        libfreetype6-dev \
-        libpng12-dev \
-        libboost-thread-dev \
-        libboost-system-dev \
-        libboost-signals-dev \
-        libboost-serialization-dev \
-        libboost-regex-dev \
-        libboost-python-dev \
-        libboost-log-dev \
-        libboost-iostreams-dev \
-        libboost-locale-dev \
-        libboost-filesystem-dev \
-        libboost-date-time-dev \
-        libboost-test-dev \
+        libpng-dev \
+        libboost-thread1.62-dev \
```

Comment:
It's probably a good idea to document at the begin of the statement why you have chosen to use boost 1.62 instead of the default boost 1.65 of Ubuntu 18.04.

Reply:
It was noted here that 1.65 is too high: https://github.com/freeorion/freeorion-sdk/issues/37#issuecomment-385667614

Reply:
Appended commit.